### PR TITLE
Use textfield to show time zone in My Account details view

### PR DESCRIPTION
**Pull request by karthikbtf**
- to be consistent with other views

Terraware App 2023-01-12 11-44-22

### CODE REVIEW

**Review by constanzauanini — APPROVED** [review]
Thanks for updating this!

**Review by charlie-tf — COMMENTED** [review]
Couple questions. But looks good!

File: src/components/MyAccount/index.tsx [hunk 1]
```diff
@@ -335,12 +339,24 @@ const MyAccountContent = ({
...
-                    onTimeZoneSelected={onTimeZoneChange}
-                    selectedTimeZone={record.timeZone}
-                    disabled={!edit}
-                  />
+                <Grid xs={12} marginLeft={3}>
+                  {edit ? (
+                    <TimeZoneSelector
+                      onTimeZoneSelected={onTimeZoneChange}
+                      selectedTimeZone={record.timeZone}
+                      disabled={!edit}
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/MyAccount/index.tsx:347]: good catch.. yes,

File: src/components/MyAccount/index.tsx [hunk 2]
```diff
@@ -335,12 +339,24 @@ const MyAccountContent = ({
...
+                      tooltip={strings.MY_ACCOUNT_TIME_ZONE_TOOLTIP}
+                    />
+                  ) : (
+                    <TextField
+                      label=''
+                      id='timezone'
+                      type='text'
+                      value={tz.longName}
+                      tooltipTitle={record.timeZone ? '' : strings.DEFAULT_TIME_ZONE_TOOLTIP}
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/MyAccount/index.tsx:356]: will make it undefined.. `''` seems to work as well but maybe for clarity undefined is better

**Pull request merged**